src/App.js

```diff
@@ -1,206 +1,52 @@
 import React from "react";
 import "./App.css";
+import products from "./state.js"
+import Header from "./components/Header";
+import Footer from "./components/Footer";
+import ImageSlider from "./components/ImageSlider";
+import SubHeader from "./components/SubHeader";
+import TopNav from "./components/TopNav";
+import ProductDetail from "./components/ProductDetail";
+
 
 function App(props) {
+	let filteredProducts = products.products.filter((product) => {
+		return product.category === props.currentCategory;
+	});
+	let filteredProductDetail = [];
+	filteredProducts.map((e) => {
```

Comment:
technically this works, but you might want to study the map function a little more to make sure you are solid with how to use it and how it works